spec/factories/products.rb

```diff
@@ -6,12 +6,13 @@
     link { "MyString" }
     email { "MyString" }
     description { "MyText" }
-    clicks_cost { 1.5 }
     store
     novelty { 3 }
     category
     trait :with_image do
-      image { fixture_file_upload(Rails.root.join('spec/fixtures/files/example.jpg'), 'image/jpg') }
+      image do
+        Rack::Test::UploadedFile.new(Rails.root.join('spec/fixtures/files/example.jpg'), 'image/jpg')
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Layout/LineLength: Line is too long. [101/100]